core/shared/platform/linux-sgx/platform_internal.h

```diff
@@ -56,8 +56,7 @@ typedef int (*os_print_function_t)(const char *message);
 void
 os_set_print_function(os_print_function_t pf);
 
-char *
-strcpy(char *dest, const char *src);
+#define strcpy(dst, src) strncpy(dst, src, strlen(src))
```

Comment:
It's because librats and wamr have the same function named `strcpy` on SGX, now I have recover it in wamr and modify the Implementation of it in librats.